examples/RFM9x_receive_example.nut

```diff
@@ -0,0 +1,68 @@
+#require "RFM9x.device.nut:0.1.0"
```

Comment:
this should go after the license - the license comments may need to be changed to single line